WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -9,9 +10,13 @@ def bad_api_parameter(e)
 
   DEFAULT_API_RESULT_LIMIT = 20
 
+  def current_resource_owner
```

Comment:
Ooh! I don't know that I've ever seen `defined?` in practical use.

Is this a deliberate or accidental rejection of the `User.find_by_id(doorkeeper_token&.resource_owner_id)` pattern?

I can live with the choice to avoid the occasional query, even if I wouldn't make it myself.